coderadar-test/src/main/asciidoc/admin.adoc

```diff
@@ -12,89 +12,85 @@ v{version}, {date}
 [introduction]
 == Introduction
 
-This guide is intended for administrators and developers who want to get a coderadar server up and running.
+This guide is intended for administrators and developers who want to get a Coderadar server up and running.
 
 == Getting coderadar up and running
-A coderadar distribution contains a WAR (Java Web Archive) file and a properties
+A Coderadar distribution contains a WAR (Java Web Archive) file and a properties
 file that contains the default configuration.
 
 === Configuration Parameters
 
-WARNING: This documentation is work in progress. If you want to contribute, have a look at this task in the
- issue tracker: https://github.com/reflectoring/coderadar/issues/146.
+Coderadar can be configured by adjusting the parameters in the local.application.properties file (either before jar-packaging or by supplying the file alongside the jar).
 
-=== Starting coderadar standalone
+==== Configuring the scan interval
 
-WARNING: This documentation is work in progress. If you want to contribute, have a look at this task in the
- issue tracker: https://github.com/reflectoring/coderadar/issues/147.
+`coderadar.scanIntervalInSeconds=30`
 
-=== Deploying coderadar on an application server
+Adjusting this value will change how often projects are checked for new commits/branches.
 
-WARNING: This documentation is work in progress. If you want to contribute, have a look at this task in the
- issue tracker: https://github.com/reflectoring/coderadar/issues/148.
+==== Configuring the working directory
 
-== Clustering
+This is the directory project will be cloned to. It's path can be adjusted by changing the following property:
```

Comment:
This is the directory **projects** will be cloned to.

Reply:
Done